sklearn/covariance/tests/test_covariance.py

```diff
@@ -196,6 +192,18 @@ def test_ledoit_wolf():
     assert(lw.precision_ is None)
 
 
+def test_ledoit_wolf_large():
```

Comment:
I have played a bit with that example. The following runs in 300ms:

```
diff --git a/sklearn/covariance/tests/test_covariance.py b/sklearn/covariance/tests/test_covariance.py
index 329f37d..094c641 100644
--- a/sklearn/covariance/tests/test_covariance.py
+++ b/sklearn/covariance/tests/test_covariance.py
@@ -192,15 +192,16 @@ def test_ledoit_wolf():
     assert(lw.precision_ is None)
 
 
-def test_ledoit_wolf_large():
-    # test that ledoit_wolf doesn't error on "large" data
+def test_ledoit_wolf_wide():
+    # test that ledoit_wolf doesn't error on data with a "large" number of
+    # features
     rng = np.random.RandomState(0)
-    X = rng.normal(size=(10, 1002))
+    X = rng.normal(size=(20, 500))
     lw = LedoitWolf().fit(X)
-    assert_almost_equal(lw.covariance_, np.eye(1002), 0)
+    assert_almost_equal(lw.covariance_, np.eye(500), 1)
     cov = lw.covariance_
 
-    lw = LedoitWolf(block_size=1024).fit(X)
+    lw = LedoitWolf(block_size=128).fit(X)
     assert_almost_equal(lw.covariance_, cov)
```

also the by doubling the number of samples, the estimated covariance almost equals the identity on the first decimal. I also changed the `block_size` to a much smaller number to make sure that it does not impact the result. `block_size=10` also works but is slightly slower.